### ISSUE TITLE
override languages specified in translation file

### ISSUE THREAD

**Issue by drewwells** [futuweb/webpack-amdi18n-loader#6]
Our main/root translation file is controlled by a third-party, so it shouldn't be modified. In that file, all languages are activated. Is it possible to override that via amdi18n options?

**Comment by drewwells** [comment]
Possibly related, the modules bundled by this loader seem to be invisible to webpack. It would be helpful if the languages could be optionally loaded or dynamically loaded later.

Given the example on readme.js, how would I configure webpack 2 to ignore bundling zh-cn language file? When using a regular require, I don't see any hints that extra files are present in the emitted bundle but at 486kb this is much larger than the file being listed.

```
chunk    {0} 0.bundle.js, 0.bundle.js.map 486 kB {1} [rendered]
   [80] ../javascripts/vendor/oj/v2.2.0/resources/nls/ojtranslations.js 486 kB {0} [built]
```
```
15K ../javascripts/vendor/oj/v2.2.0/resources/nls/ojtranslations.js
```

**Comment by TooBug** [comment]
I'll consider if it's possible to override the root lang file.

The key point is that the language files are consumed by the loader itself (using `fs` module) , rather than webpack. So you can not see any info printed by webpack. In other worlds, the language files are not controlled by webpack.

Maybe it's good to add some query to do this?

```javascript
require('amdi18n?root=./otherroot&disable=[zh-cn,jp]!mylang');
```

Under consideration, Any advise?

**Comment by drewwells** [comment]
That's a good idea! It might be useful to add an enable option too &enable=[es] to whitelist which languages are loaded.

Disable would be very handy on its own. The library I'm consuming is sending metadata through the root file which causes build errors: https://github.com/oracle/oraclejet/blob/master/dist/js/libs/oj/resources/nls/localeElements.js

`....,__noOverlay:!0,__defaultNoOverlayLocale:"en-US"});`

**Comment by TooBug** [comment]
`enable` and `disable` query is shipped in 0.5.0.
